specification/network/resource-manager/Microsoft.Network/stable/2022-05-01/azureFirewall.json

```diff
@@ -1148,6 +1208,117 @@
         }
       },
       "description": "List of SNAT IP Prefixes learnt by firewall to not SNAT"
+    },
+    "FirewallPacketCaptureParametersFormat": {
+      "type": "object",
+      "properties": {
+        "durationInSeconds": {
+          "type": "string",
+          "description": "Duration of packet capture in seconds."
+        },
+        "numberOfPacketsToCapture": {
+          "type": "string",
```

Comment:
should these be int ?

Reply:
yes, let me fix this. 
Will also need to move the PR to new api version. Will get this ARM review accepted and the nwill port over to new 2022-07-01 API